javascript/timeHandler.js

```diff
@@ -152,7 +152,7 @@ function updateTimers() {
 	
 	var timeFrom = Math.floor((new Date).getTime() / 1000);
 	
-	$$(".gameTimeRemaining .timeremaining").map(function(c) {
+	$$(".timeremaining").map(function(c) {
```

Comment:
What is the need for removing the .gameTimeRemaining? The reason I ask is because anytime we have to change the js files it can cause problems. We do somewhat address this by having a file, global\definitions.php, that has a js version in it to help force cache busting, but some phone and browser configurations still keep the js files cached, and having the wrong version will throw errors for people. If this change is needed, include the definitions.php file with the js version incremented. As a tip too, we do the same thing with css so if you ever have to change css, you can cache bust it in the same file.

Reply:
IIRC, this is because there is a .timeremaining which is not a child of .gameTimeRemaining which also needs to be decrementing correctly. (I believe it's the timer until the phase swaps).

Reply:
I've incremented the version.